docs/components_page/components/collapse/accordion.py

```diff
@@ -39,7 +39,7 @@ def toggle_accordion(n1, n2, n3, is_open1, is_open2, is_open3):
     ctx = dash.callback_context
 
     if not ctx.triggered:
-        return ""
+        return ['']*3
```

Comment:
I think what I originally meant to put here was `button_id = ""` rather than `return ""`, which results in none of the if statements triggering and you hit the `return False, False, False` at the end. Can you either go with that or `return False, False, False` (since `is_open` expects a boolean).

Reply:
Yes sure